server/RenderContext.cs

```diff
@@ -626,9 +626,37 @@ public void Render(XGraphics graphics)
                     }
                 }
                 timers.Add(new Timer("droyne"));
-#endregion
+                #endregion
+
+                #region droyne
+                //------------------------------------------------------------
+                // Non-Hiver Worlds
+                //------------------------------------------------------------
+                if (styles.nonHiverWorlds.visible)
+                {
+                    solidBrush.Color = styles.nonHiverWorlds.textColor;
+                    foreach (World world in selector.Worlds)
+                    {
+                        if (world.HasCode("Nh"))
+                        {
+                            string glyph = "\u2605";
```

Comment:
Maybe a different glyph? I'm partial to ✻ (U+273B TEARDROP-SPOKED ASTERISK) for Hiver things...